src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -80,7 +80,7 @@ internal sealed class PackageInfo : NamedItemInfo, IPackageInfo {
                 try {
                     var result = await _host.Session.PackagesFunctionsNamesAsync(Name, REvaluationKind.BaseEnv);
                     functions = result.Children<JValue>().Select(v => (string)v.Value);
-                } catch (TaskCanceledException) { } catch (REvaluationException) { }
+                } catch (RException) { }
```

Comment:
And it won't. One of the reasons why `RHostDisconnectedException` is derived from `OperationCanceledException` is because `DoNotWait` automatically swallows task cancellation, and so it will swallow `RHostDisconnectedException`. So until we don't expect `rm(list = ls(all = TRUE))` to fail with `REvaluationException`, `DeleteAllVariablesCommand` is fine.